modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APIEndpointRelevantObjectEntryModelListenerTest.java

```diff
@@ -533,6 +555,36 @@ public void test() throws Exception {
 				"headless-builder/endpoints", Http.Method.POST
 			).toString(),
 			JSONCompareMode.STRICT);
+		JSONAssert.assertEquals(
+			JSONUtil.put(
+				"status", "BAD_REQUEST"
+			).put(
+				"title",
+				"There is an API schema that is not scoped like its related " +
```

Comment:
Not sure if this description error is confusing :thinking: 

What about "The API Endpoint and the API Schema must be scoped by the same element" @4lejandrito any idea??

Reply:
`The API Endpoint and the API Schema must be have the same scope`

Reply:
Must have*

I like It! Thanks!

Reply:
Me too :smile:

Reply:
:roll_eyes: